contracts/ERC721Safe.sol

```diff
@@ -9,6 +9,13 @@ contract ERC721Safe {
     // ERC721 contract => amount of tokens owned by Safe
     mapping(address => uint) public _balances;
 
+    function fundERC721(address tokenAddress, address owner, uint tokenID) public {
+        IERC721 erc721 = IERC721(tokenAddress);
+        erc721.transferFrom(owner, address(this), tokenID);
```

Comment:
Looking at `fundERC20` as well. Whats the real difference between this and `lockERC721`? Trying to understand the use case.

Reply:
fund is called externally to find the bridge (`address(this)`), lock is used on deposits and takes a recipient as a param. lock is not public, fund is